srcpkgs/libpinyin/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'libpinyin'
+pkgname=libpinyin
+version=2.2.1
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr"
+hostmakedepends="intltool libtool pkg-config autoconf-archive autoconf automake wget"
+makedepends="db-devel libdb-cxx gnome-common glib-devel glib gettext-devel"
+short_desc="Support library for PinYin and ZhuYin (Bopofomo)"
+maintainer="Ben Sung Hsu <pobetiger+github@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/libpinyin/libpinyin"
+distfiles="https://github.com/libpinyin/libpinyin/archive/${version}.tar.gz"
+checksum="38a434f5f29607dcdf7c721cdff5a2428f03297a99064c7e15a9a64c1b831069"
+
+pre_configure() {
+	./autogen.sh ${configure_args}
```

Comment:
hmmm, no, autogen.sh should never run configure-steps, that is reserved for do_configure() phase.

Reply:
Not sure how to fix this one, the project's autogen.sh runs configure...